code/public/index.html

```diff
@@ -13,6 +12,11 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
+    <link
+      href="https://fonts.googleapis.com/css?family=Inconsolata&display=swap"
+      rel="stylesheet"
+    />
+    <link href="./Menlo-Regular.ttf" rel="stylesheet" />
```

Comment:
I get an error that this file doesn't exist. Have you tried using `@font-face` instead when including a font?